devtools/create_project/xcode/create_project.xcodeproj/project.pbxproj

```diff
@@ -180,6 +180,7 @@
 		F9A66C2E1396D36100CEE494 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CLANG_CXX_LANGUAGE_STANDARD="c++0x";
```

Comment:
I don't understand why c++0x?

Reply:
That's what I found in create_project itself, when it generates a project for Xcode. I really have no clue about Xcode. Would you prefer c++11? Change it also in create_project?

Reply:
we have now c++11 required, so I think yes